.dependabot/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
```

Comment:
It should actually be version 2: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates

Reply:
Using the dependabot.yml, one cannot automerge updates that do not affect package.json file. The option to automerge can be achieved using the config.yml that has an option to automerge  updates that affect only the package-lock.json and create PRs for version bumps that affect the package.json.  see https://dependabot.com/docs/config-file/

Reply:
The documentation from dependabot.com is the old version 1. But dependanbot moved to github and won't be developed further as version 1 :-)

If I remember correct, automerging is done by other github actions when using dependabot version 2. The merging itself is not an issue for now but only what changes we get. We only want updates that are related to the package.json instead of the package-lock.json.

Reply:
sorry my bad.. I will read the documents again to see how we can achieve that using version 2.

Reply:
@Ruben Hoping that I have not missed anything, I think dependabot does not offer this posibility as per now. 
The versioning strategy documentation is not really very clear about
using 'widen' or 'increase' and I beleive it does not guarantee that the PRs of updates that only affect the package.jsonfile will be created, rather it will be in addition to those that are in package-lock.json.
I suggest we try  setting the versioning strategy to widen or increase using a target branch and monitor the pull requests that will be generated.